static/js/authentication/controllers/register.controller.js

```diff
@@ -41,6 +42,11 @@
                             $mdToast.showSimple('Email with an activation link has been sent.');
                             $state.go('auth.login');
                         }, function (data, status) {
+                            
+                            if (status === 403) {
```

Comment:
alright, status seems to be undefined, don't use that then just do `data.status`